src/v2/view-builder/views/email/AuthenticatorEmailViewUtil.js

```diff
@@ -9,32 +9,42 @@ const CheckYourEmailTitle = View.extend({
   },
   template: hbs`
       {{#if email}}
-        {{i18n 
-            code="oie.email.verify.alternate.magicLinkToEmailAddress" 
-            bundle="login" 
-            arguments="email" 
-            $1="<span class='strong'>$1</span>"
-        }}
+        {{#if secondaryEmail}}
+          {{i18n
+              code="oie.email.verify.alternate.magicLinkToEmailAddress.with.secondary.email"
+              bundle="login"
+              arguments="email;secondaryEmail"
+              $1="<span class='strong'>$1</span>"
```

Comment:
Not part of this change, but I think we should no-translate attribute here as well.

Reply:
Done.